packages/IMAPClient-Core.package/ICFolder.class/instance/recursiveChildFolders.st

```diff
@@ -0,0 +1,9 @@
+accessing
+recursiveChildFolders
+	
+	| childFolderCollection |
+	
+	childFolderCollection := self childFolders collect: [:child | child recursiveChildFolders].
```

Comment:
I think the variable could be removed, it doesn't add information for the reader. Do a one-liner instead